edge-modules/DirectMethodSender/Settings.cs

```diff
@@ -25,22 +25,22 @@ public class Settings
                     configuration.GetValue<string>("TargetModuleId", "DirectMethodReceiver"),
                     configuration.GetValue<TransportType>("TransportType", TransportType.Amqp_Tcp_Only),
                     configuration.GetValue<TimeSpan>("DirectMethodDelay", TimeSpan.FromSeconds(5)),
-                    configuration.GetValue<Uri>("AnalyzerUrl", new Uri("http://analyzer:15000")));
+                    Option.Maybe(configuration.GetValue<Uri>("AnalyzerUrl")));
```

Comment:
is it safer to check "AnalyzerUrl" is not empty or white spaces?

Reply:
It will throw an exception if it tries to make a Uri from an empty string or null. This seems fine to me as it is not a valid value for Uri.